tools/mason/MasonUpdate.chpl

```diff
@@ -153,10 +153,17 @@ proc updateRegistry(tf: string, args: list(string)) {
   checkRegistryChanged();
   for ((name, registry), registryHome) in zip(MASON_REGISTRY, MASON_CACHED_REGISTRY) {
```

Comment:
We are actually already looping over `MASON_REGISTRY` tuple values, so the `name` is what we want to print. on the "Updating" lines.

Go check out the definition of `MASON_REGISTRY` (a parens-less procedure, so search `proc MASON_REGISTRY`), for reference.